tensorflow/python/ops/variable_scope_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
there is a variable scope test in kernel_tests; please put your tests there following the existing style of the file.